std/range/package.d

```diff
@@ -1228,6 +1231,382 @@ unittest
     auto c = chain(a, b);
 }
 
+
+/**
+    Choose one of multiple ranges at runtime.
+
+    The ranges may be different, but they must have the same element type. The
+    result is a range that offers the $(D front), $(D popFront), and $(D
+    empty) primitives. If all input ranges offer random access and $(D
+    length), $(D choose) offers them as well.
+
+    Params:
+        index = which range to choose, must be less than the number of ranges
+        rs = tuple of 1 or more ranges
```

Comment:
Shortening this to "1 or more ranges" is better I think. The tuple situation is so confused in D that I think this will only cause more confusion for someone reading the docs.